File: C Language/Unit 3/pro6.c
/**
    size of array
*/

#include<stdio.h>

void main(){
    int ip1[10], i;

    ip1[0] = 152;
    ip1[1] = 332;
    ip1[2] = 142;
    ip1[3] = 752;
    ip1[4] = 552;
    ip1[5] = 158;
    ip1[6] = 162;
    ip1[7] = 452;
    ip1[8] = 132;
    ip1[9] = 252;

    for(i=0; i<10; i++){
        printf("\n ip1[%d] is %d", i, ip1[i]);
    }
}

File: PHP/exception/pro6.php
<?php
    
    function division($ip1, $ip2){
        if($ip2 == 0){
            throw new Exception("Division not Possible with Zero");
        }else{
            return $ip1 / $ip2;
        }        
    }

    echo "<hr> Before Division";
    try{
        $ans = division(100, 20);
        echo "<hr> Answer is : ".$ans;
    }catch(Exception $err){
        echo "<hr> Error is : ".$err;
    }finally{
        echo "<hr> Finally Must be Executed";
        // close file / connection or cleanup code
    }
    
    echo "<hr> After Division";
?>